website-custom-views/src/content/api-reference/custom-webpack-config.mdx

```diff
@@ -37,9 +37,10 @@ module.exports = config;
 ```
 
 # Production
+
 For production mode, create a `webpack.config.prod.{js,cjs}` file and use the function `createWebpackConfigForProduction` to create and enhance the webpack configuration.
 
-The Custom Application CLI will then use the configuration exported from this file instead of the default one.
+The Custom Views CLI will then use the configuration exported from this file instead of the default one.
```

Comment:
In Custom Applications it is singular, in Custom Views, plural. Intentional?

Reply:
Good catch. I'll make singular.

Reply:
Resolved here: https://github.com/commercetools/merchant-center-application-kit/pull/3321/commits/8bf3753972b8474aa5f7f07fe103a988d59b4409